### ISSUE TITLE
Account creation on the hardware 

### ISSUE THREAD

**Issue by RobertKarp** [subor/sdk#14]
Hi, 

We have finally been able to make packages for our title, but when trying to test these on the hardware we have found that not only do the old user accounts that we had on the kits not work anymore, but we are also unable to create new ones. Please could you advise?

Thanks, 
Robert

**Comment by RobertKarp** [comment]
Hi Jake, Hope you had a good weekend. Just wanted to touch base and see if there was any progress on this?

**Comment by jake-ruyi** [comment]
We can't commit to a clear date because it's something out of our hands.  That said, there's a chance we will be back up and running the end of this week or the beginning of next.

**Comment by RobertKarp** [comment]
Hi Jake, 
Sorry i've not been in touch, is there any update on this?
Thanks, 
Robert

**Comment by jake-ruyi** [comment]
We actually brought everything back up literally yesterday.
We're still doing some tests to verify everything is working (email notifications might not be), but it should be good.
This took far longer to resolve than we expected.  But, thankfully, there shouldn't be another disruption like this for at least a few months.

**Comment by jake-ruyi** [comment]
I may have spoke too soon.  You *might* need to wait for the SDK update scheduled for Feb. 22.

TLDR; you can try it and see if it works.  If it doesn't, it will with the client we release Fed 22nd.

There aren't significant changes to the C++ API you use, so it shouldn't impact your integration.
What has changed is the registration/login portion of our main UI, now requiring a cellphone # as "highly recommended"/required by Chinese "real-name" mandate.

Normally, when we roll out an update there's both "old" and "new" server instances to cover multiple versions.  In this case everything was brought up from nothing so there's no "old" servers, just the new ones.

I say you "might" need to wait because the "current" client (the one you've got) probably still works.  The auth API is the same, we just added an additional "authentication identity" when a phone is used with the new client.  But with Chinese New Year just past we didn't get a chance to test with the previous build.

**Comment by RobertKarp** [comment]
Hi Jake, We tried with the current SDK but without success. I haven't seen a new SDK drop yet, is that still planned to come soon?
Thanks, 
Robert

**Comment by jake-ruyi** [comment]
Looks like the last blocker was resolved today, but not early enough to do a release.
We'll make another QA pass tomorrow and hopefully do the release then.  Sorry about the wait.

**Comment by jake-ruyi** [comment]
We're uploading binaries and pushing source to github at this very moment.
It's a bit rough, but things should be back to mostly working.

**Comment by jake-ruyi** [comment]
Finally uploaded.
Amongst other issues there's a bug with country codes when you logout and log back in.
Have to manually change the "+86" to "+852" (or whatever) and remove the "852" prefix.  Hopefully that makes sense:
![image](https://user-images.githubusercontent.com/38336907/53494155-0c95c680-3ad8-11e9-829f-b76385394c2c.png)

**Comment by RobertKarp** [comment]
Thanks Jake! enjoy your evening :)
